apps/web/pages/api/auth/verify-email.ts

```diff
@@ -0,0 +1,37 @@
+import type { NextApiRequest, NextApiResponse } from "next";
+import { z } from "zod";
+
+import { WEBAPP_URL } from "@calcom/lib/constants";
+import { prisma } from "@calcom/prisma";
+
+const verifySchema = z.object({
+  token: z.string(),
+});
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  const { token } = verifySchema.parse(req.query);
+
+  const foundToken = await prisma.verificationToken.findFirst({
+    where: {
+      token,
+      type: "VERIFY_ACCOUNT",
```

Comment:
Why do we need to use token type here? Isn't token unique ?

Reply:
Good point tho - it is unique on db level

Reply:
We might honestly not need it - i dont think i realised token was unique